src/ontology/mondo-edit.obo

```diff
@@ -348459,13 +348459,15 @@ synonym: "mental retardation, autosomal dominant 31" RELATED DEPRECATED [MONDO:L
 synonym: "mental retardation, autosomal dominant type 31" EXACT DEPRECATED [MONDORULE:2]
 synonym: "MRD31" EXACT ABBREVIATION [DOID:0070061, MONDO:Lexical]
 synonym: "neurodevelopmental disorder with neonatal respiratory insufficiency, hypotonia, and feeding difficulties" EXACT [OMIM:616158]
+synonym: "PURA syndrome" EXACT [https://orcid.org/0000-0001-9310-0163, PMID:28448108]
 synonym: "PURA-related severe neonatal hypotonia-seizures-encephalopathy syndrome due to a point mutation" EXACT CLINGEN_LABEL [Orphanet:438216]
 xref: DOID:0070061 {source="MONDO:equivalentTo"}
 xref: GARD:17740 {source="MONDO:GARD"}
 xref: ICD10CM:G40.4 {source="Orphanet:438216", source="Orphanet:438216/attributed", source="Orphanet:438216/ntbt"}
 xref: OMIM:616158 {source="Orphanet:438216", source="MONDO:equivalentTo", source="Orphanet:438216/e", source="DOID:0070061"}
 xref: Orphanet:438216 {source="MONDO:equivalentTo"}
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
+is_a: MONDO:1060108 {source="https://orcid.org/0000-0001-9310-0163"} ! PURA-related severe neonatal hypotonia-seizures-encephalopathy syndrome
```

Comment:
Maybe add the PMID as a source here too? It is always good to have a source in addition to one single orcid, especially for classification sources.